testsuite/tests/kuadrant/authorino/identity/token_review/test_host.py

```diff
@@ -0,0 +1,29 @@
+"""Test kubernetes token-review authorization with bound sa token that should contain host as audience by default"""
+
+import pytest
+
+
+pytestmark = [pytest.mark.authorino]
+
+
+@pytest.fixture(scope="module")
+def authorization(authorization):
+    """Add kubernetes token-review identity without any audiences"""
+    authorization.identity.add_kubernetes("token-review-host")
+    return authorization
+
+
+@pytest.fixture(scope="module")
+def service_account_token(openshift, hostname, create_service_account):
+    """Create service account and request its bound token with the hostname as audience"""
+    service_account = create_service_account("tkn-rev")
+    return openshift.get_serviceaccount_auth_token(service_account, [hostname.hostname])
+
+
+def test_host_audience(client, service_account_token):
+    """Test kubernetes token-review by adding hostname audience to the sa token and using it for the request"""
+    response = client.get("/get")
+    assert response.status_code == 401
+
+    response = client.get("/get", headers={"Authorization": "Bearer " + service_account_token})
```

Comment:
Maybe another Auth class, or using Oauth Auth class?

Reply:
I used the auth class for API keys. It provides identical functionality. Let me know what you think